ci/configure_v4.sh

```diff
@@ -11,3 +11,4 @@ env CONJUR_AUTHN_LOGIN=admin CONJUR_AUTHN_API_KEY=secret conjur policy load --as
 CONFIGURE
 
 docker cp $(docker-compose ps -q conjur_4):/opt/conjur/etc/ssl/ca.pem ./tmp/conjur.pem
+export CONJUR_SSL_CERTIFICATE="$(cat tmp/conjur.pem)"
```

Comment:
Should we move this into `test.sh` since that's where we set the var for v5? Or maybe we should move the v5 cert var config into `configure_v5.sh`? Either way, would be nice if they were consistent.

Reply:
Yeah - moved to test.sh for consistency